src/devices/xyzroe.ts

```diff
@@ -0,0 +1,274 @@
+                    const unit = data2[0];
+                    if (devid) {
+                        payload['device_' + name] = devid;
+                    }
+
+                    const valRaw = msg.data['presentValue'];
+                    if (unit) {
+                        let val = utils.precisionRound(valRaw, 1);
+
+                        const nameLookup: KeyValueAny = {
+                            'C': 'temperature',
+                            'V': 'voltage',
+                            'A': 'current',
+                            'W': 'power',
+                        };
+
+                        let nameAlt = '';
+                        if (unit === 'A') {
+                            if (valRaw < 1) {
+                                val = utils.precisionRound(valRaw, 3);
+                            } else {
+                                val = utils.precisionRound(valRaw, 2);
+                            }
+                        }
+                        nameAlt = nameLookup[unit];
+
+                        if (nameAlt === undefined) {
+                            const valueIndex = parseInt(unit, 10);
+                            if (! isNaN(valueIndex)) {
+                                nameAlt = 'val' + unit;
+                            }
+                        }
+
+                        if (nameAlt !== undefined) {
+                            payload[nameAlt + '_' + name] = val;
+                        }
+                    }
+                }
+            }
+            return payload;
+        },
+    } satisfies Fz.Converter,
+    zigusb_on_off_invert: {
+        cluster: 'genOnOff',
+        type: ['attributeReport', 'readResponse'],
+        convert: (model, msg, publish, options, meta) => {
+            if (msg.data.hasOwnProperty('onOff')) {
+                const payload: KeyValueAny = {};
+                const endpointName = model.hasOwnProperty('endpoint') ?
+                    utils.getKey(model.endpoint(meta.device), msg.endpoint.ID) : msg.endpoint.ID;
+                const state = msg.data['onOff'] === 1 ? 'OFF' : 'ON';
+                payload[`state_${endpointName}`] = state;
+                return payload;
+            }
+        },
+    } satisfies Fz.Converter,
+};
+
+
+function zigusbBtnConfigExposes(epName: string) {
+    const features = [];
+    features.push(e.enum('button_mode', exposes.access.ALL,
+        getSortedList(buttonModesList)).withEndpoint(epName));
+    features.push(e.enum('link_to_output', exposes.access.ALL,
+        getSortedList(inputLinkList)).withEndpoint(epName));
+    features.push(e.enum('bind_command', exposes.access.ALL,
+        getSortedList(bindCommandList)).withEndpoint(epName));
+    return features;
+}
+
+const definitions: Definition[] = [
+    {
+        zigbeeModel: ['ZigUSB'],
+        model: 'ZigUSB',
+        vendor: 'xyzroe',
+        description: '[Zigbee USB power monitor and switch](https://xyzroe.cc/ZigUSB)',
```

Comment:
Let's not add the link here but in the docs (zigbee2mqtt.io)

Reply:
Ok. No problem, I did it out of old habit)
But I really don’t really understand how to add a link there if the description is auto-generated.

Reply:
I tried to compile the documentation locally - but my device did not appear in the list, the MD file was not created. (indicated INCLUDE_DEVICE=ZigUSB)
I assume that while the device is not in the zigbee-herdsman-converters, it cannot be added to the documentation? Or I did something wrong.  

I'm not sure that you even need to build locally to add custom description. 
Tell me where to look please

Reply:
You can just create the `.md` file, only the non generated part has to be added (between `<!-- Notes BEGIN: You can edit here. Add "## Notes" headline if not already present. -->`)

Reply:
I read it, but description is autogenerated field. And link is located in description. I saw the same in other devices. Don't understand how to add link to description if it's autogenerated...

Reply:
I meant using `description: 'Zigbee USB power monitor and switch',` and linking to the device in the `## Notes` section (not in the description)

Reply:
But lot of devices use link exactly in description. Is that option deprecated?

Reply:
Yes, I prefer not to use it anymore